boot.go

```diff
@@ -11,11 +11,15 @@ import (
 	"github.com/deis/builder/pkg/cleaner"
 	"github.com/deis/builder/pkg/conf"
 	"github.com/deis/builder/pkg/gitreceive"
-	"github.com/deis/builder/pkg/gitreceive/storage"
 	"github.com/deis/builder/pkg/healthsrv"
 	"github.com/deis/builder/pkg/sshd"
 	"github.com/deis/builder/pkg/sys"
 	pkglog "github.com/deis/pkg/log"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	_ "github.com/docker/distribution/registry/storage/driver/azure"
+	"github.com/docker/distribution/registry/storage/driver/factory"
+	_ "github.com/docker/distribution/registry/storage/driver/gcs"
+	_ "github.com/docker/distribution/registry/storage/driver/s3-aws"
```

Comment:
> That plan is fine with me for now. FWIW, my guess is that docker/distribution#1537 is going to be a "won't fix".

Don't count us out. We'd like to avoid extra maintenance overhead of a separate project. However, if people find the model to be effective and some extra work is done to increase fitness for external usage, we will likely reconsider. There really is nothing registry specific about these packages.

> Not use anonymous imports unless for good, documented reason

These are to support back registration. If you find the documentation to be lacking, we are willing to accept a PR. Up until this point, the usage of this package has been mostly internal. If we split out this project, the back registration would likely remain in the registry project, as it is very specific to the configuration environment. There is no reason to continue to maintain it in this manner, other than convenience, so we're open to suggestions.